configuration/programs/programs-ces.csv

```diff
@@ -1,2 +1,9 @@
 Uuid,Name,Description,Void/Retire,Program concept,Outcomes concept
-${program.hearts.uuid},HEARTS,Programa HEARTS,,PIH:HEARTS Program,PIH:Generic outcome
```

Comment:
You should rename this "programs.csv" to have it override the "programs.csv" in config-pihemr, without having to remove it from config-pihemr.

Reply:
@Joseph-Developer Debes cambiar el nombre de "programs-ces.csv" a "programs.csv".